invenio_jsonschemas/config.py

```diff
@@ -44,3 +44,25 @@
 
 JSONSCHEMAS_REGISTER_ENDPOINTS_UI = True
 """Register the endpoints on the UI app."""
+
+JSONSCHEMAS_SCHEMAS = None  # loads all json schemas
+"""List of entry names to register json schemas for.
+
+If `None`, all json schemas defined through the ``invenio_jsonschemas.schemas``
```

Comment:
```suggestion
If `None`, all JSON Schemas defined through the ``invenio_jsonschemas.schemas``
```
ditto for other references